networking/network_observability/installing-operators.adoc

```diff
@@ -18,27 +18,48 @@ include::modules/network-observability-without-loki.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../networking/network_observability/configuring-operator.adoc#network-observability-enriched-flows_network_observability[Export enriched network flow data].
+* xref:../network_observability/configuring-operator.adoc#network-observability-enriched-flows_network_observability[Export enriched network flow data].
 
 include::modules/network-observability-loki-install.adoc[leveloffset=+1]
 include::modules/network-observability-loki-secret.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
-* For more information about the option to use different namespaces for the separate components, see the `spec.loki.tls.caCert.namespace` specification in the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference] and callout number 5 in the xref:../network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource].
+* For more information about the option to use different namespaces for the separate components, see the `spec.loki.tls.caCert.namespace` specification in the xref:../../networking/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference] and callout number 5 in the xref:../../networking/network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource].
 
 include::modules/network-observability-lokistack-create.adoc[leveloffset=+2]
 include::modules/network-observability-lokistack-ingestion-query.adoc[leveloffset=+2]
-include::modules/network-observability-auth-multi-tenancy.adoc[leveloffset=+1]
-include::modules/network-observability-multitenancy.adoc[leveloffset=+1]
-
-include::modules/network-observability-kafka-option.adoc[leveloffset=+1]
+include::modules/network-observability-auth-multi-tenancy.adoc[leveloffset=+2]
+include::modules/network-observability-multitenancy.adoc[leveloffset=+2]
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/77501157/c8ddc729-f5e3-4d5e-8166-b98511a70842)
@jotak

Reply:
The paragraph after "Network Observability without Loki" sounds like instead of Loki, I can replace it with explorers.  Suggest:  
  
If you only want to export flows to a Kafka consumer or IPFIX collector or you only need dashboard metrics, then you do not need to install Loki or provide storage for Loki.  Without Loki, there won't be a `Network Traffic` panel under `Observe` which means there is no overview charts, flow table, or topology.

Reply:
TY steven...I can see what you mean and I'll take the suggestion :)